src/main/java/tech/jhipster/lite/generator/setup/license/domain/LicenseModuleFactory.java

```diff
@@ -0,0 +1,33 @@
+package tech.jhipster.lite.generator.setup.license.domain;
+
+import static tech.jhipster.lite.module.domain.JHipsterModule.*;
+
+import java.time.Year;
+import tech.jhipster.lite.module.domain.JHipsterModule;
+import tech.jhipster.lite.module.domain.properties.JHipsterModuleProperties;
+
+public class LicenseModuleFactory {
+
+  public JHipsterModule buildMitModule(JHipsterModuleProperties properties) {
+    //@formatter:off
+    return moduleBuilder(properties)
+      .context()
+        .put("currentYear", Year.now().getValue())
+        .and()
+      .files()
+        .add(from("setup/license/MIT.txt.mustache"), to("./LICENSE.txt"))
```

Comment:
You can create a constant `SOURCE`:
`private static final JHipsterSource SOURCE = from("setup").append("license");`

And then use it here with the `template(String)` method that allows you to omit the `.mustache` suffix:
```suggestion
        .add(SOURCE.template("MIT.txt"), to("./LICENSE.txt"))
```

Reply:
Usually we omit `./` (same for Apache):
```suggestion
        .add(from("setup/license/MIT.txt.mustache"), to("LICENSE.txt"))
```